lua/telescope/_extensions/file_browser/actions.lua

```diff
@@ -329,8 +329,12 @@ fb_actions.remove_file = function(prompt_bufnr)
   -- format printing adequately
   print "\n"
 
-  vim.ui.select({ "Yes", "No" }, { prompt = "Remove Selected Files" }, function(_, idx)
-    if idx == 1 then
+  vim.ui.input({ prompt = "Remove selected files (Y/N): " }, function(user_answer)
```

Comment:
I commonly see the yes/no options shown with the capitalized letter indicating the default option so in this case something like `[y/N]`. Otherwise I like this change.

Reply:
Slight nitpick but I agree.